### PR TITLE
Test reboot and IceWM in gdm_session_switch

### PR DESCRIPTION
**Pull request by mdoucha**
Update gdm_session_switch test to include a reboot and IceWM. The test runs successfully on SLE 12SP2 and newer. SLE 12SP1 fails due to minor regression in IceWM but it's probably not worth fixing at this point in its lifecycle...

- Related ticket: https://progress.opensuse.org/issues/55706
- Needles: https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/1207
- Verification run:
https://openqa.suse.de/tests/3294106
https://openqa.suse.de/tests/3294107

### CODE REVIEW

File: tests/x11/gdm_session_switch.pm [hunk 1]
```diff
@@ -58,14 +77,18 @@ sub run {
...
     send_key "ret";
     assert_screen "originUser-login-dm";
     type_password;
     assert_and_click "displaymanager-settings";
     assert_and_click "dm-gnome";
     send_key "ret";
     assert_screen "generic-desktop", 120;
+
+    # Log out and switch to SLE classic
+    $self->prepare_sle_classic;
```
Comment (dzedro) [inline tests/x11/gdm_session_switch.pm]: this is done twice in this test, also on line 33-34

Comment (mdoucha) [reply tests/x11/gdm_session_switch.pm]: Yes, this is intentional test that GDM session switching still works after logging out of non-classic Gnome.

Comment (dzedro) [reply tests/x11/gdm_session_switch.pm]: Can you then update also summary, because what is there doesn't describe what is done now.

Comment (mdoucha) [reply tests/x11/gdm_session_switch.pm]: Added a line about rebooting to the summary comment in gdm_session_switch.pm.

**Pull request merged**